config/coreboot-supermicro-x11ssh-f.config

```diff
@@ -0,0 +1,888 @@
+#
```

Comment:
Please make savedefconfig and replace this file by the content of defconfig outputted.